core/src/test/kotlin/fr/sncf/osrd/sim_infra_adapter/PathPropertiesTests.kt

```diff
@@ -477,6 +485,124 @@ class PathPropertiesTests {
             )
     }
 
+    @Test
+    fun testSmallInfraSpeedLimitTagFallbacks() {
+        val rjsInfra = Helpers.getExampleInfra("small_infra/infra.json")
+        val superLowSpeedSection =
+            RJSSpeedSection(
+                "speedSection",
+                3.0,
+                mapOf(),
+                listOf(
+                    RJSApplicableDirectionsTrackRange(
+                        "TH1",
+                        ApplicableDirection.BOTH,
+                        4900.0,
+                        5000.0
+                    )
+                ),
+                null
+            )
+        rjsInfra.speedSections.add(superLowSpeedSection)
+        val infra = Helpers.fullInfraFromRJS(rjsInfra)
+        val routeName = "rt.DH2->buffer_stop.7"
+        val blocks = Helpers.getBlocksOnRoutes(infra, listOf(routeName))
+        val path =
+            makePathProps(
+                infra.blockInfra,
+                infra.rawInfra,
+                blocks.last(),
+                routes = listOf("rt.DH2->buffer_stop.7")
+            )
+
+        /*
+        On the last block, speed limit sections from small infra are
+
+        0m       100m      600m     1000m     1500m     1600m
+        |=========|=========|=========|=========|=========|
+        | 39.444  |      31.111       | 83.333  |   3.0   |     no-tag (to be respected any time)
```

Comment:
```suggestion
        | 39.444  |      31.111       | 8.333  |   3.0   |     no-tag (to be respected any time)
```

Reply:
This is correct I think: It's the default speed limit of the section, and it's 300 km/h as seen [here](https://github.com/OpenRailAssociation/osrd/blob/dd067f8d106b7006fbf5932a90676cc28ddbd718/tests/data/infras/small_infra/infra.json#L2928) (and not overridden by any other speed section).